With our Triton version the expected failure doesn't happen anymore.
See also https://github.com/pytorch/pytorch/issues/154217

Author: Alexander Grund (TU Dresden)

diff --git a/test/inductor/test_torchinductor_dynamic_shapes.py b/test/inductor/test_torchinductor_dynamic_shapes.py
index 11de2175fc4..68d33a9bb48 100644
--- a/test/inductor/test_torchinductor_dynamic_shapes.py
+++ b/test/inductor/test_torchinductor_dynamic_shapes.py
@@ -367,9 +367,6 @@ class TestInductorDynamic(TestCase):
         except Exception:
             if not expect_fail:
                 raise
-        else:
-            if expect_fail:
-                self.fail("expected to fail, but actually passed")
 
     @torch._dynamo.config.patch(
         capture_scalar_outputs=True, capture_dynamic_output_shape_ops=True
